A/Aria2/build_tarballs.jl

```diff
@@ -50,11 +50,11 @@ dependencies = [
     # `LibSSH2_jll` was compiled with.  So we explicitly select the
     # right version here.
     BuildDependency(PackageSpec(name="MbedTLS_jll", version=v"2.28")),
-    Dependency(PackageSpec(name="OpenSSL_jll"); compat = "1.1.10"),
+    Dependency(PackageSpec(name="OpenSSL_jll"); compat="1.1.10"),
```

Comment:
It's better to use OpenSSL 3 at this point
```suggestion
    Dependency(PackageSpec(name="OpenSSL_jll"); compat="3.0.8"),
```